action.yml

```diff
@@ -281,3 +279,35 @@ runs:
         phpdbg -qrr vendor/bin/phpunit --coverage-clover=coverage.xml
         ./codecov-linux -f coverage.xml;
         echo "coverage.xml generated and uploaded to codecov"
+
+    - name: "Run documentation linting"
+      if: ${{ inputs.doclinting == 'true' }}
+      shell: bash
+      run: |
+        echo "Running documentation linting"
+        if [[ ! -f vendor/silverstripe/documentation-lint/.nvmrc ]]; then
+          echo "File vendor/silverstripe/documentation-lint/.nvmrc is missing. Check dependencies."
+          exit 1
+        fi
+        # Remove any sneaky attempts to put __install-nvm.sh into pull-requests
+        if [[ -f __install-nvm.sh ]]; then
```

Comment:
Not necessary, use cp -f or equivalent

Reply:
Any discussion about this can be in https://github.com/silverstripe/gha-run-tests/pull/23#discussion_r1550757042